notebooks/search/07-inference.ipynb

```diff
@@ -0,0 +1,382 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7a765629",
+   "metadata": {},
+   "source": [
+    "# Semantic Search using the Inference API\n",
+    "\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/elastic/elasticsearch-labs/blob/main/notebooks/search/07-inference.ipynb)\n",
+    "\n",
+    "\n",
+    "Learn how to use the [Inference API](https://www.elastic.co/guide/en/elasticsearch/reference/current/inference-apis.html) for semantic search."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c99b06d",
+   "metadata": {},
+   "source": [
+    "# 🧰 Requirements\n",
+    "\n",
+    "For this example, you will need:\n",
+    "\n",
+    "- An Elastic deployment with minimum **4GB machine learning node**\n",
+    "   - We'll be using [Elastic Cloud](https://www.elastic.co/guide/en/cloud/current/ec-getting-started.html) for this example (available with a [free trial](https://cloud.elastic.co/registration?utm_source=github&utm_content=elasticsearch-labs-notebook))\n",
+    "   \n",
+    "- A paid [OpenAI account](https://openai.com/) is required to use the Inference API with \n",
+    "the OpenAI service as the OpenAI free trial API usage is limited. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "15193c10",
+   "metadata": {},
+   "source": [
+    "# Create Elastic Cloud deployment\n",
+    "\n",
+    "If you don't have an Elastic Cloud deployment, sign up [here](https://cloud.elastic.co/registration?utm_source=github&utm_content=elasticsearch-labs-notebook) for a free trial.\n",
+    "\n",
+    "- Go to the [Create deployment](https://cloud.elastic.co/deployments/create) page\n",
+    "   - Under **Advanced settings**, go to **Machine Learning instances**\n",
+    "   - You'll need at least **4GB** RAM per zone for this tutorial\n",
```

Comment:
Same here, I think there is no need to require 4GB. See how other notebooks in this directory (such as 03-ELSER) have been updated with simpler instructions.